package.json

```diff
@@ -22,6 +22,7 @@
     "build:test": "SEGMENT_HOST='https://api.segment.io' SEGMENT_WRITE_KEY='FAKE' yarn build test",
     "build:test:flask": "yarn build test --build-type flask",
     "build:test:mv3": "ENABLE_MV3=true yarn build test",
+    "build:icons": "rimraf lib build ui/components/component-library/icons/lib && svgr --out-dir ui/components/component-library/icons/lib --filename-case kebab --template ui/components/component-library/icons/template.js -- ui/components/component-library/icons/svgs && node ui/components/component-library/icons/scripts/generateIndexResources.js",
```

Comment:
Hey @brad-decker, I need some help aligning with our build conventions, where should I add this script and how should I include it in our build steps?